### PR TITLE
Break up H2 Frame Considerations

### PR DESCRIPTION
**Pull request by MikeBishop**
Splitting out Ian's suggestion to break up A.2 further.

### CODE REVIEW

File: draft-ietf-quic-http.md [hunk 1]
```diff
@@ -1926,29 +1926,55 @@ on each stream only.  As a result, if a frame type makes assumptions that frames
...
+
+HTTP/3 permits the prioritization of requests, pushes and placeholders that each
+exist in separate identifier spaces. The HTTP/3 PRIORITY frame replaces the
+stream dependency field with fields that can identify the element of interest
+and its dependency.
+
+### Header Compression Differences
+
+HPACK was designed with the assumption of in-order delivery. A sequence of
+encoded header blocks must arrive (and be decoded) at an endpoint in the same
```
Comment (LPardue) [inline draft-ietf-quic-http.md]:
```suggestion
encoded header block fragments must arrive (and be decoded) at an endpoint in the same
```

Comment (martinthomson) [reply draft-ietf-quic-http.md]: The concept of header block fragments is an h2 thing.  h3 provides the entire header block in one go, more or less, right?

Comment (LPardue) [reply draft-ietf-quic-http.md]: yeah I was looking at this from the wrong perspective. Header blocks is fine, I take back the suggestion.

**Comment by ianswett** [comment]
I thought this was generally a useful PR and it doesn't(yet) have conflicts.  Maybe take another look and ensure it still reads well to you and then merge it?

**Pull request merged**